backtrace-library/src/androidTest/java/backtraceio/library/SettingAttributesTest.java

```diff
@@ -205,4 +206,11 @@ public void uncaughtException(Thread t, Throwable e) {
             Assert.fail(exception.getMessage());
         }
     }
+
+    @Test
+    public void ensureClientCustomAttributesPassedToNewBacktraceAttributes() {
+        BacktraceClient client = new BacktraceClient(context, backtraceCredentials, clientAttributes);
+        BacktraceAttributes attributes = new BacktraceAttributes(context, null, clientAttributes);
+        assertEquals(clientAttributes.get("custom-client-attribute-id"), attributes.attributes.get("custom-client-attribute-id"));
```

Comment:
Why you're testing if you can get an attribute instead of test also the value of attribute? This method I believe, can return null which might be unexpected behaviour.

Reply:
This tests that the value of `clientAttributes["custom-client-attribute-id"] == attributes.attributes["custom-client-attribute-id"]`. I will add null checks too to make it more robust.

Reply:
but also can we be sure that `clientAttributes["custom-client-attribute-id"]` store value that it suppose to store?

Reply:
Will do, thank you!